apps/src/templates/progress/SendLessonDialog.jsx

```diff
@@ -53,24 +66,15 @@ export default class SendLessonDialog extends Component {
           style={styles.button}
           onClick={() => copyToClipboard(this.props.lessonUrl)}
         />
-        {i18n.sendLessonCopyLink()}
+        <span style={styles.buttonLabel}>{i18n.sendLessonCopyLink()}</span>
       </div>
     );
   }
 
   renderShareToGoogleRow() {
     return (
-      <div>
-        {/* TODO: Replace placeholder button with actual implementation */}
-        <Button
-          text=""
-          icon="users"
-          iconStyle={styles.buttonIcon}
-          color="white"
-          style={styles.button}
-          onClick={() => {}}
-        />
-        {i18n.sendLessonShareToGoogle()}
+      <div style={styles.row}>
+        <GoogleClassroomShareButton height={48} url={this.props.lessonUrl} />
```

Comment:
small thing: I would probably put the button heights in a constant, since we want the to be the same height and they are defined in different places

Reply:
you could just use `styles.button.height` here, yeah?